lib/dal-test/src/schemas/test_exclusive_schema_swifty.rs

```diff
@@ -43,6 +45,16 @@ pub async fn migrate_test_exclusive_schema_swifty(ctx: &DalContext) -> BuiltinsR
     // Author Resource Payload Func
     let resource_payload_to_value_func = build_resource_payload_to_value_func().await?;
 
+    // Build CodeGen Func
+    let codegen_fn_name = "test:generateCode";
+    let codegen_func_code = "async function main(input: Input): Promise < Output > {
```

Comment:
We'll need to make sure the handler is always `main` for your function, but I don't think we need to add any additional checks or anything.